application/models/Question.php

```diff
@@ -1661,7 +1661,7 @@ public function getEmptySubquestion()
     {
         $question = new Question();
         $question->qid = 0;
-        $question->title = (SettingsUser::getUserSettingValue('subquestionprefix', App()->user->id) ?? 'SQ') . '001';
+        $question->title = (Yii::app()->getConfig('subquestionprefix', 'SQ')) . '0001';
```

Comment:
I would use one less 0 in general, for answer options and questions, because more than 999 questions /answer options is very unlikely.